assets/javascript/javascript.js

```diff
@@ -77,6 +77,7 @@ $(document).ready(function() {
                     url: queryURL,
                     headers: {
                         "Authorization": "Bearer " + hashParams.access_token
+                        // bearer is standered for passing in tokens
```

Comment:
@Jenny-Le comments should always go above, or beside the line of code that it relates to.

Ex.
```
// bearer is standered for passing in tokens
"Authorization": "Bearer " + hashParams.access_token
```
or
```
"Authorization": "Bearer " + hashParams.access_token // bearer is standered for passing in tokens
```